ebu_tt_live/bindings/validation/timing.py

```diff
@@ -5,6 +5,30 @@
 from ebu_tt_live.strings import ERR_SEMANTIC_VALIDATION_TIMING_TYPE
 import itertools
 
+
+def EBUTTDRegionsOverlap(region1, region2):
```

Comment:
Not sure if were following a specific style but should this function name start with a lower case? And could it not be part of the TimingValidationMixin class?

Reply:
I thought about making it a class function, but it would have resulted in more verbose code, and is more generally useful as as standalone method. Arguably, if anything, it should be a member of the `d_region_type`, e.g. `d_region_type.overlaps(other_region)` but I didn't think of that at the time.

Reply:
Actually that's such a good idea I now feel obliged to refactor it!

Reply:
Done in 81e6593, with further improvement in 61291b3 if you want to look again.